lib/npm.js

```diff
@@ -90,7 +90,12 @@ NPM.prototype.listDependencies = function listDependencies(obj, name, prefix) {
         };
     }
 
-    var fileName = name + '-' + newObj.version + '.tgz';
+    // npm pack will remove @ symbols from the name.
+    var santitizedName = name.replace(/[@]/, '');
```

Comment:
Why /[@]/ ? This regex matches only the first @ sign as you are not using global flag (is this intentional) as I think the only place where an @ sign can appear in a package name is in the beginning.
Even so, why not simply /@/ ?